src/helpers/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const CONTRACT_TX_ID = 'w5ZU15Y2cLzZlu3jewauIlnzbKw-OAxbN9G5TbuuiDQ'
+export const VITE_OTHENT_API_KEY = import.meta.env.VITE_OTHENT_API_KEY
```

Comment:
Will we need to remove ENV vars when the time comes to upload statically to Arweave?